Symfony/CS/Fixer/Contrib/HeaderCommentFixer.php

```diff
@@ -91,7 +91,7 @@ private static function encloseTextInComment($header)
         foreach ($lines as $line) {
             $comment .= rtrim(' * '.$line)."\n";
         }
-        $comment .= " */\n";
+        $comment .= ' */';
```

Comment:
I was just saying that surely this change was already made on the 1.7 branch, so it doesn't need fixing again?